src/router/metadata/metadataHandler.ts

```diff
@@ -43,7 +43,7 @@ export default class MetadataHandler implements Capabilities {
     }
 
     async capabilities(request: CapabilitiesRequest): Promise<GenericResponse> {

```

Comment:
Can we add two test case for this with `productInfo`? One test case in which all of productInfo's  mandatory fields are defined and one in which the fields are not defined?

Reply:
@nguyen102 Only the `orgName` field remains mandatory after these changes, as was previously the case. Do you still want me to add the extra two test cases?

Reply:
I think adding a test case for having ALL params filled out and one with just the required ones, will suffice. So just adding 1 test case for ALL params in productInfo

Reply:
Alright, finally got back around to this one. Added 1 new test case to cover when ALL param in productInfo are present.